### PR TITLE
Add Sphinx site and shields; update CI Pipeline

### PR DESCRIPTION
**Pull request by umarcor**
This PR brings a minimal plumbing to this repo.

The main motivation is to have some site on GitHub Pages, which we can cross-reference from [edaa-org.github.io](https://edaa-org.github.io) instead of having broken links.
By the way, a "work in progress" sign is added both in the README and in the index of the site.

A subset of the Pipeline from pyTooling/Actions is used to replace the CI jobs. Unit testing and coverage are disabled because this repo does not have tests yet.

### CODE REVIEW

File: .github/workflows/Pipeline.yml [hunk 1]
```diff
@@ -1,274 +1,106 @@
...
-          - {python: 3.6, icon: 🔴}
-          - {python: 3.7, icon: 🟠}
-          - {python: 3.8, icon: 🟡}
-          - {python: 3.9, icon: 🟢}
+  Params:
+    uses: pyTooling/Actions/.github/workflows/Parameters.yml@r0
+    with:
+      name: pyEDAA.OutputFilter
+      python_version: '3.9'
+      python_version_list: '3.6 3.7 3.8 3.9'
```
Comment (Paebbels) [inline .github/workflows/Pipeline.yml:18]: Use default and enable 3.10 too.

File: .github/workflows/Pipeline.yml [hunk 2]
```diff
@@ -1,274 +1,106 @@
...
+      - BuildTheDocs
+      - PublishToGitHubPages
+    with:
+      package: ${{ fromJson(needs.Params.outputs.params).artifacts.package }}
+      remaining: |
+        ${{ fromJson(needs.Params.outputs.params).artifacts.doc }}
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.6
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.7
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.8
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.9
```
Comment (Paebbels) [inline .github/workflows/Pipeline.yml:105]: Add 3.10 here too.

File: .github/workflows/Pipeline.yml [hunk 3]
```diff
@@ -1,274 +1,106 @@
...
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.coverage }}
```
Comment (Paebbels) [inline .github/workflows/Pipeline.yml:106]: Missing line break according to GH online diff view.

File: .github/workflows/Pipeline.yml (hunk 1 above)
Comment (umarcor) [reply .github/workflows/Pipeline.yml]: Done.

File: .github/workflows/Pipeline.yml (hunk 2 above)
Comment (umarcor) [reply .github/workflows/Pipeline.yml:104]: Done.

File: .github/workflows/Pipeline.yml (hunk 3 above)
Comment (umarcor) [reply .github/workflows/Pipeline.yml]: Done.

File: .github/workflows/Pipeline.yml (hunk 1 above)
Comment (Paebbels) [reply .github/workflows/Pipeline.yml]: Why not use no key at all, so we can globally update via pyTooling/Actions?

The other repositories use this feature.

Comment (umarcor) [reply .github/workflows/Pipeline.yml]: Done.

**Pull request merged**